src/index.html

```diff
@@ -1,13 +1,40 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <title>Product cards</title>
+    <link rel="stylesheet" href="./style.css">
+  </head>
+  <body>
+    <main>
+      <div class="product">
+        <div class="product__card" data-qa="card">
+          <div class="product__image">
+            <img class="product__img" src="./images/imac.jpeg" alt="Mac Pic">
+          </div>
+          <div class="product__info">
+            <h3 class="product__title">Моноблок APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)</h3>
+            <p class="product__code">Код товара: 195434</p>
+            <div class="product__rating">
+              <div class="product__stars ">
+                <div class="stars"></div>
+                <div class="stars"></div>
+                <div class="stars"></div>
+                <div class="stars"></div>
+                <div class="stars"></div>
+              </div>
+              <span class="product__feedback">Отзывов: 5</span>
+            </div>
+            <div class="product__price">
+              <span class="text">Цена:</span>
+              <span class="value">69 999 грн</span>
```

Comment:
text and value - redundant BEM blocks
you should create correct card element classes